clients/fossid-webapp/src/main/kotlin/FossIdRestService.kt

```diff
@@ -159,17 +163,11 @@ interface FossIdRestService {
         fun create(url: String, client: OkHttpClient? = null): FossIdServiceWithVersion {
             logger.info { "The FossID server URL is $url." }
 
-            val minReadTimeout = Duration.ofSeconds(60)
-            val retrofit = Retrofit.Builder()
-                .apply {
-                    client(
-                        client?.run {
-                            takeUnless { readTimeoutMillis < minReadTimeout.toMillis() }
-                                ?: newBuilder().readTimeout(minReadTimeout).build()
-                        } ?: OkHttpClient.Builder().readTimeout(minReadTimeout).build()
-                    )
-                }
-                .baseUrl(url)
+            val retrofit = Retrofit.Builder().apply {
+                client(
+                    (client?.newBuilder() ?: OkHttpClient.Builder()).addInterceptor(TimeoutInterceptor).build()
```

Comment:
Nit: Could be shortened to `(client ?: OkHttpClient()).newBuilder()`.

Reply:
If you look at the implementations of `newBuilder()` and `OkHttpClient.Builder()`, the latter does less and therefore saves some CPU cycles :grin:

Reply:
I know, but it's shorted and aligns with existing code. But I'm ok either way.